templates/CRM/Tag/Form/Edit.tpl

```diff
@@ -58,14 +58,14 @@
         {/if}
     </table>
     {else}
-        <div class="status">{ts 1=$delName}Are you sure you want to delete <b>%1</b>?{/ts}<br />{ts}This tag will be removed from any currently tagged contacts, and users will no longer be able to assign contacts to this tag.{/ts}</div>
+        <div class="status">{if !empty($delName)}{ts 1=$delName}Are you sure you want to delete <b>%1</b>?{/ts}<br />{ts}This tag will be removed from any currently tagged contacts, and users will no longer be able to assign contacts to this tag.{/ts}{/if}</div>
```

Comment:
I've only quickly skimmed this PR and ignored the price ones but this one jumps out - seems like this _should_ be a fail if it's empty. In fact if it's empty it should wipe your database and tell you to start again (grin).